src/nextra-theme-docs.css

```
@@ -2958,3 +2958,12 @@ kbd._border._gap-1 {
     color: hsl(var(--color-neu-500));
   }
 }
+
+:root {
+  --nextra-bg: 251, 251, 249;
+}
+@media (prefers-color-scheme: dark) {
+  :root {
+    --nextra-bg: 15, 15, 12;
+  }
+}
```

App-router and pages-router routes seem to be happy with this simple change.

```suggestion
.dark {
    --nextra-bg: 15, 15, 12;
}
```